cmd/incus/storage_bucket.go

```diff
@@ -3,13 +3,17 @@ package main
 import (
 	"fmt"
 	"io"
+	"net/url"
 	"os"
+	"path"
 	"sort"
 	"strings"
+	"time"
 
 	"github.com/spf13/cobra"
 	"gopkg.in/yaml.v2"
 
+	incus "github.com/lxc/incus/client"
```

Comment:
We usually to not set the name for this import, so should just be `github.com/lxc/incus/client`

Reply:
Still an issue :)